pom.xml

```diff
@@ -74,6 +74,10 @@
         <!-- revision variable is set by automation tools. dev-SNAPSHOT is the
         default, local value. Please don't change it. -->
         <revision>dev-SNAPSHOT</revision>
+
+        <splunk.token></splunk.token>
```

Comment:
a short comment will help fellow developers to understand what is it for :)

Reply:
Yep, comments added.